reana_commons/api_client.py

```diff
@@ -78,7 +79,6 @@ class JobControllerAPIClient(BaseAPIClient):
     def submit(
         self,
         workflow_uuid="",
-        experiment="",
```

Comment:
Hmm, it looks good to remove the good old `experiment`, although being on `maint-0.7`, it may be a bit dangerous to change API.  Considering there are no other applications, I guess we might be OK here